src/evaluator.rs

```diff
@@ -1,68 +1,28 @@
-use base64;
 use charset::{self, Charset};
-use quoted_printable;
-use std::{error, fmt, result, str};
 
 use crate::parser::{Ast, Node::*};
 
-#[derive(Debug)]
+#[derive(thiserror::Error, Debug)]
```

Comment:
In their doc, they use straight `Error` instead of qualifying it. So I guess I would go with `Error` directly, what do you think?

Reply:
If we'd use `Error` straight, then we'd have a conflict with our `Error` enum here.

Reply:
In their doc they also prefix errors with the domain and not `Error`. But it's ok, I'm ok with that!

Reply:
> In their doc they also prefix errors with the domain and not Error. But it's ok, I'm ok with that!

hm... if it'll show up as something bad, then we can fix that quickly ^^